### ISSUE TITLE
Re-assign keybinding for open detected link or go to command so both can be invoked from within the accessible buffer

### ISSUE THREAD

**Issue by meganrogge** [microsoft/vscode#188101]
cc @jooyoungseo, @Tyriar

**Comment by jooyoungseo** [comment]
Great idea. The only concern is that presenting both info in one quick pick might be too messy and cause unnecessary confusion.

**Comment by meganrogge** [comment]
Yeah @jooyoungseo already said it's tough to navigate the quickpick as is. Nice idea, but I fear that would complicate things

**Comment by Tyriar** [comment]
@meganrogge do the headers in the quick pick not work well in screen readers? After filtering should we be announcing something like x items in header1, y items in header2?

**Comment by meganrogge** [comment]
An issue is that it wraps around when using arrow keys. So, for example, using the link quick pick is odd because you can wrap around to some pretty irrelevant "word link" results quite easily.

**Comment by meganrogge** [comment]
We could have this be one command but make the user select which type they're interested in IE `links` or `symbols` as the first step

**Comment by Tyriar** [comment]
Configuring on first use like that can be a bad experience as you don't know what you want initially and may not know how to change it after. Perhaps there should be a filter for workspace search links, like `.+[/\\].+`, and/or we hide non-relevant looking ones unless you hit show more links.

**Comment by meganrogge** [comment]
Oh, I didn't mean configuring. I meant showing that option every time the command is used.

**Comment by meganrogge** [comment]
IMO, these should remain separate commands because the point of a quick pick is you know what you're getting and don't have to go looking for a particular category

**Comment by sandy081** [comment]
Can you please provide steps to verify?
